sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/FileSourceCodecSuite.scala

```diff
@@ -57,12 +57,7 @@ class ParquetCodecSuite extends FileSourceCodecSuite {
   override val codecConfigName: String = SQLConf.PARQUET_COMPRESSION.key
   // Exclude "lzo" because it is GPL-licenced so not included in Hadoop.
   override protected def availableCodecs: Seq[String] =
-    if (System.getProperty("os.arch") == "aarch64") {
-      // Exclude "brotli" due to PARQUET-1975.
-      Seq("none", "uncompressed", "snappy", "lz4", "gzip", "zstd")
-    } else {
-      Seq("none", "uncompressed", "snappy", "lz4", "gzip", "brotli", "zstd")
-    }
```

Comment:
BTW, @gengliangwang . Could you add some comment? Otherwise, someone may want to try to re-add it in the future.

Reply:
+1